src/components/table/tableDirective.ts

```diff
@@ -23,6 +23,7 @@ export interface ITableScope extends angular.IScope {
   rows: ITableRowScope[];
   tableType: string;
   tableTypeClass: string;
+  selectedItems: any;
```

Comment:
Please add documentation in the JSDOC for this interface to document this property.